replication/row_event.go

```diff
@@ -856,6 +877,39 @@ type RowsEvent struct {
 	ignoreJSONDecodeErr     bool
 }
 
+// enum class enum_row_image_type { WRITE_AI, UPDATE_BI, UPDATE_AI, DELETE_BI };
```

Comment:
Does AI and BI represent "after image" and "before image"? A bit unfamiliar

Reply:
Yes, I preserve naming from [mysql sources](https://github.com/mysql/mysql-server/blob/1bfe02bdad6604d54913c62614bde57a055c8332/sql/rpl_record.h#L39). I can rename it for more clarity if you think so.

Reply:
I can see their meanings in the linked MySQL code now 😄